model/src/main/proto/topic.proto

```diff
@@ -249,3 +252,35 @@ message CellularDataPreference {
   // Preference on whether to use cellular data
   bool use_cellular_data = 2;
 }
+
+// Corresponds to a single subtopic that can be trained, or reviewed.
+message Subtopic {
+  // The ID of the subtopic.
+  string subtopic_id = 1;
+
+  // The title of the subtopic.
+  string title = 2;
+
+  // The list of skill ids associated with each of the subtopic.
```

Comment:
Update:
`// The list of skill ids associated with the subtopic.`

Reply:
updated.